MandyC6420/safespot
# Ac safespot 1

Description
Authentication/Registration completed.  Safe spot cards able to be added/posted all state is being saved correctly

Added functionality to GET, POST,  Safe Spots from the safespot API. 

Related Ticket(s)
https://trello.com/c/VbvRiAeE/2-1-authentication-and-login
https://trello.com/c/e0FBxOBk/5-3-creating-and-removing-a-safe-spot

Steps to Test
Clone down Safe Spot
npm start command at root 
start api with:  json-server -p 8088 -w database.json

Once opened please click on Locations
On Locations you will see an affordance to add a location
Once selected a form field is displayed to add a Safe Spot to the database

Checklist:
 I have commented my code, particularly in hard-to-understand areas
 I have made corresponding changes to the documentation
 My changes generate no new errors
 I have added tests that prove my fix is effective or that my feature works
 New and existing unit tests pass locally with my changes
